src/Test/TestGameEvent.java

```diff
@@ -354,6 +355,56 @@ boolean testResupplyEvent()
     // Clean up.
     testMap.removeUnit(apc);
     testMap.removeUnit(mech);
+    testMap.removeUnit(mech2);
+    testMap.removeUnit(recon);
+
+    return testPassed;
+  }
+
+  private boolean testUnitJoinEvent()
+  {
+    boolean testPassed = true;
+
+    // Add some units.
+    Unit mech = addUnit(testMap, testCo1, UnitEnum.MECH, 1, 4);
+    Unit mech2 = addUnit(testMap, testCo1, UnitEnum.MECH, 1, 5);
+    mech.initTurn(testMap);
+    mech2.initTurn(testMap);
```

Comment:
I typically like to use more descriptive names for the unit variables - it makes it easier to autocomplete them and to tell what's supposed to be going on at a glance. This goes double for when you actually break a test: "What's a mech2, and why should I care about it?"
"recipient" and "donor" could work?

Reply:
Works for me.